app/views/online-class/scheduler.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <page-template>
+    <div class="scheduler">
+      <div class="header">
+        {{ header }}
+      </div>
+      <div class="body">
+        <class-info
+          v-if="step === 'class'"
+          :code-language-map="codeLanguageMap"
+          :levels="levels"
+          @change-class-info="changClassInfo"
+        />
+        <available-time
+          v-else-if="step === 'available'"
+          :class-info="classInfo"
+          @back="step = 'class'"
+          @next="selectTime"
+        />
+        <student-info
+          v-else-if="step === 'student'"
+          :class-details="classDetails"
+          @change-student-info="changeStudentInfo"
+          @back="step = 'available'"
+        />
+        <next-step
+          v-else
+        />
+      </div>
+    </div>
+  </page-template>
+</template>
+
+<script>
+import moment from 'moment'
+import pageTemplate from '../parents/PageTemplate.vue'
+import classInfo from './components/classInfo.vue'
+import availableTime from './components/availableTime.vue'
+import studentInfo from './components/studentInfo.vue'
+import nextStep from './components/nextStep.vue'
+
+import { tempBookTime, bookTime } from '../../core/api/online-classes'
+export default {
+  name: 'SchedulerView',
+  components: {
+    pageTemplate,
```

Comment:
why did you with naming vue files with this convention of `abcDef`? Most places, we use 'AbcDef'. We don't have a rule that enforces it, but is this convention used anywhere else in the code?

Reply:
ok, i can update them. just my habit 😂